central/policy/service/validator.go

```diff
@@ -185,6 +185,13 @@ func (s *policyValidator) validateEventSource(policy *storage.Policy) error {
 			}
 		}
 	}
+
+	if s.isNodeEventPolicy(policy) {
+		if !booleanpolicy.ContainsAllOf(policy, booleanpolicy.FileAccess) {
```

Comment:
This check might be better as - should contain atleast one node event field - we should consider a runtime type Nodeevent in the field_metadata.go.